replicator/src/main.rs

```diff
@@ -86,9 +70,9 @@ fn main() {
         Keypair::new()
     };
 
-    let storage_mining_pool_pubkey = matches
-        .value_of("storage_mining_pool_pubkey")
-        .map(|value| value.parse::<Pubkey>().unwrap());
+    let storage_mining_pool_pubkey = "StorageMiningPoo111111111111111111111111111"
```

Comment:
what's this for?  if auto-redeeming, just put this in the code that generates the instruction vector...